Remove @param and @returns tags if they have no description

deyaaeldeen opened the pull request:
Address feedback for https://github.com/Azure/azure-sdk-for-js/pull/12912.

ramya-rao-a commented on sdk/core/core-http/src/credentials/accessTokenRefresher.ts:
```diff
@@ -34,7 +32,6 @@ export class AccessTokenRefresher {
    * then requests a new token,
    * then sets this.promise to undefined,
    * then returns the token.
-   * @param options -
```
There are many more occurrences like this in the repo. Look for `@param .* -$` after enabling the regex search in your editor

ramya-rao-a commented on sdk/tables/data-tables/src/TableBatch.ts:
```diff
@@ -246,7 +246,6 @@ function parseBatchResponse(batchResponse: HttpOperationResponse): TableBatchRes
 /**
  * Prepares the operation url to be added to the body, removing the SAS token if present
  * @param url - Source URL string
- * @returns
```
There are many more occurrences for this. Look for `@returns$` after enabling regex and excluding the `arm-*` folders

deyaaeldeen commented:
@ramya-rao-a I addressed your feedback. I also decided to replace all occurrences of `@ignore` with `@hidden` instead of asking everyone to do it which could have been confusing.
